argoverse/utils/transform.py

```diff
@@ -25,7 +25,9 @@ def quat2rotmat(q: np.ndarray) -> np.ndarray:
     Returns:
         R: Array of shape (3, 3) representing a rotation matrix.
     """
-    assert np.isclose(np.linalg.norm(q), 1.0, atol=1e-12)
+    if not np.isclose(np.linalg.norm(q), 1.0, atol=1e-12):
```

Comment:
Should we log this so the user knows? Additionally, is it okay if we fail if they pass in all zeroes for coefficients?

Reply:
it's logged now